src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -26,12 +33,16 @@ public class UserAffiliationServiceImpl implements UserAffiliationService {
   private final UserTenantService userTenantService;
   private final TenantService tenantService;
   private final KafkaService kafkaService;
+  private final FolioExecutionContext folioExecutionContext;
+  private final FolioModuleMetadata folioModuleMetadata;
   private final ObjectMapper objectMapper = new ObjectMapper();
 
   @Override
   @SneakyThrows
   @Transactional
   public void createPrimaryUserAffiliation(String eventPayload) {
+    FolioExecutionContext requestContext = (FolioExecutionContext) folioExecutionContext.getInstance();
+    String currentTenantId = folioExecutionContext.getTenantId();
```

Comment:
please define variables in place where they're used. Here it's used only in "else" block